### PR TITLE
Fix NXP-14020 mp-upgrade snapshots policy

### PR DESCRIPTION
**Pull request by yjulienne-nuxeo**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/nuxeo/nuxeo/pull/241" target="_blank">ReviewNinja

File: nuxeo-distribution/nuxeo-launcher/src/test/java/org/nuxeo/launcher/TestConnectBroker.java [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the GNU Lesser General Public License
+ * (LGPL) version 2.1 which accompanies this distribution, and is available at
+ * http://www.gnu.org/licenses/lgpl-2.1.html
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
```
Comment (efge) [inline nuxeo-distribution/nuxeo-launcher/src/test/java/org/nuxeo/launcher/TestConnectBroker.java]: This should be the Apache License, not LGPL.

File: nuxeo-distribution/nuxeo-launcher/src/test/resources/log4j.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
   <appender name="CONSOLE" class="org.apache.log4j.ConsoleAppender">
     <errorHandler class="org.apache.log4j.helpers.OnlyOnceErrorHandler" />
     <param name="Target" value="System.out" />
-    <param name="Threshold" value="WARN" />
+    <!-- <param name="Threshold" value="WARN" /> -->
```
Comment (efge) [inline nuxeo-distribution/nuxeo-launcher/src/test/resources/log4j.xml]: Please don't commit changes to the log4j configuration. Log output should be empty when running tests.

File: nuxeo-distribution/nuxeo-launcher/src/test/resources/packages/store/A-1.0.0/package.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<package type="addon" name="A" version="1.0.0">
+  <title>Package A</title>
+  <description>
+    <p>Description of A</p>
+  </description>
+  <home-page>http://doc.nuxeo.com/x/7YGo</home-page>
```
Comment (jcarsique) [inline nuxeo-distribution/nuxeo-launcher/src/test/resources/packages/store/A-1.0.0/package.xml]: why https://doc.nuxeo.com/display/NXDOC/Nuxeo+Diff ? :p

**Comment by jcarsique** [comment]
:+1:

File: nuxeo-distribution/nuxeo-launcher/src/test/java/org/nuxeo/launcher/TestConnectBroker.java (hunk 1 above)
Comment (yjulienne-nuxeo) [inline nuxeo-distribution/nuxeo-launcher/src/test/java/org/nuxeo/launcher/TestConnectBroker.java]: done in next commit

File: nuxeo-distribution/nuxeo-launcher/src/test/resources/log4j.xml (hunk 2 above)
Comment (yjulienne-nuxeo) [inline nuxeo-distribution/nuxeo-launcher/src/test/resources/log4j.xml]: done in next commit

File: nuxeo-distribution/nuxeo-launcher/src/test/resources/packages/store/A-1.0.0/package.xml (hunk 3 above)
Comment (yjulienne-nuxeo) [inline nuxeo-distribution/nuxeo-launcher/src/test/resources/packages/store/A-1.0.0/package.xml]: copy-paste left overs... but not relevant

**Pull request merged**